### ISSUE TITLE
Volume Reconstruction

### ISSUE THREAD

**Issue by nist-anka** [IbisNeuronav/Ibis#155]
When you have more than 1 acquisition and reconstruct volumes in one ibis session, the second and next volumes are not properly reconstructed.
You have to reconstruct one volume, save scene, close ibis, start ibis and reconstruct next volume. The order of reconstruction is not important. If you have 2 acquisitions a1 and a2, you reconstruct first a1, you cannot reconstruct a2. You reconstruct first a2, you caannot reconstruct a1.

**Comment by drouin-simon** [comment]
Are you working on this bug? I think this would be high priority.

**Comment by nist-anka** [comment]
I am working on it, but it is hopeless.  My knowledge of itk is limited. My
knowledge of OpenCL nul. I managed to check that the bug is really in
reconstruction, not in interface.
When I run it in valgrind, to check memory and uninitialized variables, it
works. I still think, it is initialization problem or cleaning between 2
sequences.
At the same time I corrected some memory leaks reported by valgrind and
made a pr 157. !56 is for API and it may wait.
Anka
